doc/source/index.rst

```diff
@@ -1,51 +1,47 @@
-Welcome to PDFmorph's Documentation!
+Welcome to SNMF's Documentation!
 ====================================
 
-``PDFmorph``: An easy way to compare PDFs.
+``SNMF``: Determine structural phase components, their ratios, and how they change with time from
+sets of PDFs or powder diffraction patterns.
 
 Introduction
 ++++++++++++
 
-``PDFmorph`` is a Python package that increases the insight 
+``snmf`` is a Python package that increases the insight
```

Comment:
please rewrite first making a general comment that could apply to any signal, and then a particular one related to the diffraction problem we focused on.  See my example above.

Reply:
I will do this